src/amo/components/AddonsCard/index.js

```diff
@@ -63,10 +74,13 @@ export default class AddonsCard extends React.Component {
       }
     }
 
+    const allClassNames = makeClassName('AddonsCard', className, {
+      'AddonsCard--horizontal': type === 'horizontal',
+    });
     return (
       <CardList
         {...otherProps}
-        className={makeClassName('AddonsCard', `AddonsCard--${type}`, className)}
```

Comment:
The old component was adding `AddonsCard--link` by default but I can't find any references to that class anywhere. I think it had bitrotted. The only one in use appears to be `AddonsCard--horizontal`.